package.json

```diff
@@ -10,14 +10,14 @@
     "lint": "eslint src/scripts",
     "coverage": "nyc --reporter=lcov --reporter=text --reporter=text-summary mocha",
     "bundlesize": "bundlesize",
-    "cypress:run": "$(npm bin)/cypress run",
-    "cypress:open": "$(npm bin)/cypress open",
+    "cypress:run": "cypress run",
+    "cypress:open": "cypress open",
     "cypress:ci": "cypress run --record --group --ci-build-id $GITHUB_SHA",
     "test": "run-s test:unit test:e2e",
     "test:unit": "mocha",
     "test:unit:watch": "mocha --watch --inspect=5556",
     "test:e2e": "run-p --race start cypress:run",
-    "js:watch": "NODE_ENV=development node server.js",
+    "js:watch": "cross-env NODE_ENV=development node server.js",
```

Comment:
I think instead of introducing new dependency you just can remove `NODE_ENV=development` everywhere in the `package.json`. `server.js` actually checks `NODE_ENV !== 'production'`, so, undefined will works as well as `development` here.

PS: If you also can add `NODE_ENV: production` to the [`cypress.yml` action env](https://github.com/jshjohnson/Choices/blob/15d54c7d342f5e44ad2ccf8a3ec5b81e56752a4c/.github/workflows/cypress.yml#L26) as part of removing this - that will be 💯